src/_guides/language/effective-dart/style.md

```diff
@@ -323,6 +315,16 @@ defaultTimeout
 kDefaultTimeout
 {% endprettify %}
 
+### DON'T explicitly name libraries
+
+Appending a name to the `library` directive is technically possible,
```

Comment:
Do you think we should add a version note to this entry?

Reply:
Ahh.... well you know I'm not a fan of [version notes](https://github.com/dart-lang/site-www/issues/4309) and I really don't want to keep adding them to the docs until we codify some rhyme and reason to them. 

Let's revisit version-noting this and other 2.19 doc updates next month when addressing the [What's New and Language Evolution pages](https://github.com/dart-lang/site-www/issues/4404) for this release. (I'll make a note on that issue to make sure we address this comment specifically)

I don't have a solid answer right now but I'm imaging a "lighter" way to do it than what we're currently doing, but I need to do some research into how others handle this in their docs.